### PR TITLE
Allow fetching ExecuteResponse for invocation actions when only actionDigest is known

### PR DESCRIPTION
**Pull request by bduffany**
If we only have an `actionDigest` in the URL (i.e. when clicking on an action in the cache requests card), look up the Execution from the executions table using the invocation ID + action digest, and fetch the ExecuteResponse from the resulting execution ID. (We store the ExecuteResponse in AC at the key given by `hash(executionID)`)

**Related issues**: Fixes https://github.com/buildbuddy-io/buildbuddy-internal/issues/3454

### CODE REVIEW

File: app/invocation/invocation_action_card.tsx [hunk 1]
```diff
@@ -241,23 +304,33 @@ export default class InvocationActionCardComponent extends React.Component<Props
...
-          this.fetchStdout(actionResult);
-          this.fetchStderr(actionResult);
-          this.fetchServerLogs(executeResponse);
-        }
+        return executeResponse;
+      });
+  }
+
+  fetchExecuteResponseByActionDigest(actionDigest: build.bazel.remote.execution.v2.Digest) {
+    return rpc_service.service
```
Comment (jdhollen) [inline app/invocation/invocation_action_card.tsx:312]: this should be using the "regional" rpc service via `getRegionalServiceOrDefault`, right?

File: server/buildbuddy_server/buildbuddy_server.go
Comment (jdhollen) [inline server/buildbuddy_server/buildbuddy_server.go:1]: this is just an accidental carry-over from the other pr, right?

Comment (bduffany) [reply server/buildbuddy_server/buildbuddy_server.go:1]: woops, rebased

File: app/invocation/invocation_action_card.tsx (hunk 1 above)
Comment (bduffany) [reply app/invocation/invocation_action_card.tsx]: yup, done

**Review by jdhollen — APPROVED** [review]

**Pull request merged**